### ISSUE TITLE
Alacritty Crashes When Resizing Window

### ISSUE THREAD

**Issue by lpaulino07** [alacritty/alacritty#6150]
Alacritty will crash if you try to resize it on Windows 11.  I'm not sure if this is a Windows or Alacritty issue. I'm assuming it's the former but I'll report it here just in case

### System
OS: Windows 11 22H2
Version: alacritty 0.10.1

### Log
A part of 'alacritty print-events' log until crash happens: [print-events.txt](https://github.com/alacritty/alacritty/files/8972301/print-events.txt)

**Comment by chrisduerr** [comment]
Is there an actual crash message?

**Comment by lpaulino07** [comment]
> Is there an actual crash message?

**Comment by lpaulino07** [comment]
> Is there an actual crash message?

Oops closed by mistake. No, it doesn't show any crash message

**Comment by lpaulino07** [comment]
Here is a video reproducing the crash: 

https://user-images.githubusercontent.com/31266959/175470553-4c05d382-780c-4d07-a47e-4237bd6d9dc6.mp4

Also, I am running Beta AMD GPU drivers (22.5.2) so I'm not sure if that's messing up with the OGL backend.

**Comment by chrisduerr** [comment]
Does it ever actually crash the process completely? Looks like it's still going.

**Comment by lpaulino07** [comment]
That's because I have the parent running --print-events so it stays open after the child instance closed, but if I have only one alacrity window open then the process disappears after crash

**Comment by chrisduerr** [comment]
> That's because I have the parent running --print-events so it stays open after the child instance closed, but if I have only one alacrity window open then the process disappears after crash

And the command actually returns? With what exit code? Does it print the `Goodbye` message, indicating success?

**Comment by lpaulino07** [comment]
It just stays like the last second of the video forever, nothing else is printed by the parent window.

**Comment by chrisduerr** [comment]
You just said that if you have only one Alacritty window open, the process disappears though, right? If the process disappears, the process is obviously done and thus it won't stay frozen.

**Comment by lpaulino07** [comment]
I may have misunderstood your question above, so I'll just try to explain everything in one message so that it's hopefully easier to understand. 
When using only one alacritty window and it crashes by resizing it, the window closes and the process disappears. I'm able to create another instance without issue. 
When I'm using the setup showed in the video in order to see the log messages and it crashes, the last message printed on the parent window is "Width: X, Height: Y" and the child window closes. The parent window is not frozen and it can be used normally.

**Comment by chrisduerr** [comment]
Can you output the log to a different terminal, so it crashes AND you get the output and exit code?

**Comment by lpaulino07** [comment]
Not sure if this is what you mean. But I ran it from windows powershell and the same thing happened.

https://user-images.githubusercontent.com/31266959/175752508-3f5895ef-7cdf-48a0-959f-522c8de440e9.mp4

**Comment by chrisduerr** [comment]
The process still seems to be attached to the powershell instance though, since no prompt is showing? Though that could just be Windows being daft. Unfortunately without a crash log I'm not sure this is due to Alacritty or Windows 11 just doing something silly. You'd likely have to attach a debugger to get a backtrace.

**Comment by lpaulino07** [comment]
I've never used a debugger before so I'm not sure if I'm doing this right. I downloaded wingdb and attached the alacritty process. Then I used the "run" command to restart it so that it runs with gdb from the beginning. As soon as the app restarted I got this error:
 `Error 20 (this feature has not been implemented yet) in function AVolute::GetProductInfoT <lambda_3920e95365a48b95dd51020986e9e351>::operator ()`
Then after crashing it by resizing I got this message: 
`Thread 1 received signal SIGSEGV, Segmentation fault. 0x00007ffbe14b6af0 in atio6axx!DrvPresentBuffers ()                                                                     from C:\WINDOWS\System32\DriverStore\FileRepository\u0379774.inf_amd64_6708f07e8af3f5cf\B379425\atio6axx.dll`
After that I ran the command "backtrace" from gdb and got a bunch of messages, then ran "step" a couple times until the app closed. I attached the log file from gdb: 
[gdb.txt](https://github.com/alacritty/alacritty/files/8985945/gdb.txt)
